### ISSUE TITLE
A11y: [Be in your element] Buttons have barely perceivable keyboard focus ind...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-android#6396]
- **Checkpoint ID:** 11.1.4.11/1.1
- **Component:** Be in your element

### Issue Description
Buttons have barely perceivable keyboard focus indicators.

### Recommendation
Increase focus state contrast to at least 3:1.

--- 
*Report Date: 25.02.2026*

**Comment by bmarty** [comment]
@gaelledel we need an overlay color for button with Black BG.

<img width="342" height="180" alt="Image" src="https://github.com/user-attachments/assets/c1396010-c449-48d3-a21e-98c4bd6b388b" />

<img width="329" height="189" alt="Image" src="https://github.com/user-attachments/assets/849d2c14-46c7-42f4-9d20-94e11667846e" />

**Comment by bmarty** [comment]
Fixed by #7182

<img width="803" height="487" alt="Image" src="https://github.com/user-attachments/assets/d2a8ae56-44ba-45b2-bfc5-5e95c300e79b" />
